samples/benchmarks/coremark/Kconfig.sysbuild

```
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "share/sysbuild/Kconfig"
+
+config IMAGE_2_BOARD
+	string "Second additional board name to build sample for"
+	default "nrf5340dk_nrf5340_cpunet" if $(BOARD) = "nrf5340dk_nrf5340_cpuapp"
+	default "nrf5340dk_nrf5340_cpunet" if $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns"
+
+config IMAGE_3_BOARD
+	string "Third additional board name to build sample for"
```

Image numbering is very unintuitive. Can we rely on already existing configurations and use image names instead of relying on raw image numbers?

https://github.com/nrfconnect/sdk-nrf/blob/main/samples/Kconfig#L34

---

but different SoCs have different names. It was discussed with Vova and this was found as best solution. Please suggest better solution if you want to change it.

---

sure. With the following definition, you are not married to any particular target but you can support any DK built on top of nRF53 SoC (thingy53, nRF53 DK, nRF53 Audio DK)

```suggestion
config APP_TARGET_CPUNET
	string "Additional target for SoC with Network CPU (cpunet)"
	depends on DOMAIN_CPUNET_BOARD
	default DOMAIN_CPUNET_BOARD
```

---

Discussed offline. @MarekPieta proposed solution. Will add it so you can see if it is ok for you.